boards/pyportal/examples/blinky_interrupt.rs

```diff
@@ -0,0 +1,163 @@
+//! Turn on and off with an LED
```

Comment:
This could potentially be moved to another PR. I wrote this while debugging my interrupt code, and thought it might be a useful example for others in the future.